edb/lang/graphql/types.py

```diff
@@ -89,8 +106,13 @@ def __init__(self, edb_schema):
 
         self._gql_schema = GraphQLSchema(
             query=query,
-            types=[objt for name, objt in self._gql_objtypes.items()
-                   if name != 'Query'],
+            types=[
+                objt for name, objt in
+                sorted(itertools.chain(self._gql_objtypes.items(),
+                                       self._gql_inobjtypes.items()),
+                       key=lambda x: x[0])
+                if name != 'Query'
+            ],
```

Comment:
This is a bit hard to read, I'd suggest to extract `sorted(...)` into a separate assignment statement `types = sorted(...)`.

Reply:
From the performance point of view it makes more sense to first filter types and sort them later.

Reply:
OK, I've refactored the code a little bit to make it less obscure.